test/all_tests.jl

```diff
@@ -81,6 +81,8 @@ end
     # solve and check output is expected for each input json file
     @testset "$j" for j in json_names
         output = JSON3.read(run_solve(read_json("inputs", j)))
+        @info output
+        #isdefined(Main, :Infiltrator) && Main.infiltrate(@__MODULE__, Base.@locals, @__FILE__, @__LINE__)
```

Comment:
remove comment

Reply:
and the `@info`

Reply:
oh my goodness. thought i 've removed them. :)

Reply:
done